packages/theme-chalk/src/notification.scss

```diff
@@ -37,8 +37,7 @@
 
   @include e(content) {
     font-size: $--notification-content-font-size;
-    line-height: 21px;
-    margin: 6px 0 0 0;
+    line-height: 24px;
```

Comment:
```suggestion
    line-height: 24px;
    margin: 6px 0 0 0;
```

I think the margin style should be retained. When title is none, it should be overwritten with `margin:0;`.

So could you modify the `index.vue` of notification to implement it?

Reply:
Agreed

Reply:
you mean just change the line-height to 24px?but i try if it has 6px top margin, it will be this.
I don't know why margin should be retained.Can you explain it in detail?
![image](https://user-images.githubusercontent.com/32129111/104912526-55eea800-59c7-11eb-9c80-9ff7d449c65e.png)

Reply:
What @Ryan2128 meant was, using javascript to determine whether notification had content or not. When no content is presented, you need to set the `margin` to `0` via javascript, so that you do not need to remove that line of `margin: 6px 0 0`

Reply:
Something like 

```vue
<!-- pseudo code -->
<template>
  <title :class="{has-body: !!$slots.body}">
    <icon />
  </title>
</template>
<style>
  .has-body {
     margin: 0;
  }
</style>
```

Reply:
Oh, I know what that means? You want to keep this margin to make it more spaced when it has a title, right. I've taken this into consideration before, but I don't think it's necessary to make such a fuss.like this.
margin: 0
![image](https://user-images.githubusercontent.com/32129111/104914654-70765080-59ca-11eb-952e-39e430f3ed5b.png)
margin: 6px 0 0 0
![image](https://user-images.githubusercontent.com/32129111/104914680-78ce8b80-59ca-11eb-9daf-6b617fb85ea7.png)

Reply:
so I won't change it, you can fix it by your own, it is easy.I'm a minimalist.